HTTP2-Core.package/H2PaddedFrame.class/class/optPaddedOffset.st

```diff
@@ -0,0 +1,3 @@
+constants
+optPaddedOffset
```

Comment:
Möglichst keine Abkürzungen verwenden; ich weiß, dass die Kategorie Klassenseitig gelöscht wurde, aber weiterhin die Instanzmethode verbleibt, die diesen Namen trägt. Warum nennen wir diese nicht auch einfach paddedOffset?

Reply:
In diesem Fall kommt das von @Hyumiris, da es ein optionaler Padding ist. Vielleicht würde optionalPaddedOffset ausgeschrieben besser sein?